src/components/Board.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState, useEffect } from 'react'
+import Square from './Square'
+import lines from './winning-conditions'
+import Confetti from 'react-dom-confetti'
+import confettiConfig from './confetti-config'
+
+//Board props
+type BoardType = {
+  squares: string[][]
```

Comment:
it makes sense to create an actual type for your game state

```suggestion
  squares: GameState[]
```

in src, you can create a typings folder. Inside of that folder, I'd create a file index.d.ts (typescript definition file). Any type you add in here will be available globally in your project

```ts
type GameState = string[]
```